api/v1/controllers/aspects.js

```diff
@@ -18,6 +18,8 @@ const doDeleteAllAssoc =
                     require('../helpers/verbs/doDeleteAllBToMAssoc');
 const doDeleteOneAssoc =
                     require('../helpers/verbs/doDeleteOneBToMAssoc');
+const doPostAssoc =
+                    require('../helpers/verbs/doPostBtoMAssoc');
 const doFind = require('../helpers/verbs/doFind');
```

Comment:
@iamigo thanks a lot. 😅 Was looking at it for quite some time but was not able to figure it out.